titan-web-client/src/layouts/dashboard/components/sidebar/Sidebar.js

```diff
@@ -67,6 +68,21 @@ class Sidebar extends React.Component {
         {/* /> */}
         {/* </SidebarContentGroup> */}
 
+        <WithAcl
+          options={['titan.user:canAckEventExcuse']}

```

Comment:
With the new ACL changes that have landed, this will change to something like `mod.titan:canAckEventExcuse`.